amundsen_application/api/jira/v0.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from http import HTTPStatus
+from flask import Response, jsonify, make_response, request
+from flask.blueprints import Blueprint
+from flask import current_app as app
+from typing import Any
+
+from amundsen_application.jira.jira_client import JiraClient
+from amundsen_application.api.utils.request_utils import get_query_param
+
+LOGGER = logging.getLogger(__name__)
+
+jira_blueprint = Blueprint('jira', __name__, url_prefix='/api/jira/v0')
+
+
+@jira_blueprint.route('/issues', methods=['GET'])
+def get_jira_issues() -> Response:
+    """
+    Given a table key, returns all JIRA tickets containing that key. Returns an empty array if none exist
+    :return: List of JIRA tickets
+    """
+    try:
+        table_key = get_query_param(request.args, 'key', 'Request requires a key')
+        missing_config_response = validate_jira_properties()
+        if len(missing_config_response) > 0:
+            return make_response(jsonify({'msg': missing_config_response}), HTTPStatus.NOT_IMPLEMENTED)
+
+        jira_client = JiraClient()
```

Comment:
I wonder if client needs to be created everytime the API is called. 
e.g: https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/api/table.py#L119

Reply:
yeah i didn't think it was a good pattern but i wasn't sure how to maintain "state" in a flask endpoint. do you have any advice on that?

Reply:
Yeah, Flask endpoint can be a instance like this one:
https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/api/table.py#L113
https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/__init__.py#L85

Reply:
Actually question, does this seem like a good tradeoff? Since the existing app is not restful, and adding in one rest endpoint seems to be sort of weird given that it's not part of a bigger refactor.

Reply:
I also think that because the table key contains // and / this actually isn't a good pattern to follow - ie  'hive://gold.redshift/dimension_applicants' causes routing errors. 
given that for a get it should be like issues/hive://gold.redshift/dimension_applicants which won't map to a url. Would like to chat in person about this :)

Reply:
Added resource endpoints - this did require renaming part of a blueprint as the flask restful Api class is named Api automatically and caused collisions with our existing /api/ blueprint endpoint.